bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/WeatherCapability.java

```diff
@@ -29,15 +30,16 @@
  *
  */
 @NonNullByDefault
-public class WeatherCapability extends RestCapability<WeatherApi> {
+public class WeatherCapability extends BufferedWeatherCapability {
+    private final static int MIN_DATA_VALIDITY_S = 3;
```

Comment:
And in that case, isn't 1 second too low ?

Reply:
You're correct - let's cut the pear in two :-)